workflow-service/src/test/java/com/redhat/parodos/workflow/execution/controller/WorkFlowControllerTest.java

```diff
@@ -49,7 +53,7 @@ public void ExecuteWithoutAuth() throws Exception {
 	public void ExecuteWithValidData() throws Exception {
 
 		// given
-		WorkFlowRequestDTO workFlowRequestDTO = new WorkFlowRequestDTO().builder().build();
+		WorkFlowRequestDTO workFlowRequestDTO = WorkFlowRequestDTO.builder().build();
```

Comment:
Why invoke build() and then use setters?

Reply:
Existing code but fixed.